.gitlab-ci.yml

```diff
@@ -38,23 +37,35 @@ build:
 
 test:acceptance:
   stage: test
-  image: docker:18-dind
+  image: docker:19.03-dind
```

Comment:
I prefer running dind as a service rather than using the image (e.g. as we do in `mender-qa/gitlab-pipeline/stage/build.yml`) it self even if you're just using `docker:19.03`, so you don't have to do the rather awkward startup waiting :point_down:

Reply:
I also prefer that, but is not possible :) This job uses integration tests framework, so it needs the same kind of setup that we have for [integration tests](https://github.com/mendersoftware/mender-qa/blob/master/gitlab-pipeline/stage/test.yml#L613).